projects/plugins/crm/admin/automations/main.page.php

```diff
@@ -12,10 +12,6 @@
 
 defined( 'ZEROBSCRM_PATH' ) || exit;
 
-if ( ! apply_filters( 'jetpack_crm_feature_flag_automations', false ) ) {
-	return;
-}
-
```

Comment:
I'm removing this because it didn't have any value/impact besides running the filter one more time, unnecessarily.

_It's not needed since the logic that registers the page isn't added unless the feature flag is enabled._